bin/functional-tests/test_astronomer.py

```diff
@@ -36,8 +36,15 @@ def test_prometheus_user(prometheus):
     assert user == "nobody", \
         f"Expected prometheus to be running as 'nobody', not '{user}'"
 
-def test_houston_test_fixture(houston_api):
-    data = houston_api.check_output("whoami")
+def test_houston_config(houston_api):
+    """ Make assertions about Houston's configuration
+    """
+    data = houston_api.check_output("echo \"config = require('config'); console.log(JSON.stringify(config))\" | node -")
```

Comment:
@adam2k @schnie ty for your help on this bit of shell-ism. Should be easy now to add just one more test for other houston config things in future.